app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1554,14 +1557,16 @@ public void onNewMessages(Message[] messages) {
     }
 
     Snackbar snackbar;
-
+    CoordinatorLayout.LayoutParams snackBarParams;
     private void setupSnackBar() {
         final CoordinatorLayout coordinatorLayout = (CoordinatorLayout) findViewById(R.id.coordinatorLayout);
         snackbar = Snackbar.make(coordinatorLayout, "", Snackbar.LENGTH_LONG);
         View v = snackbar.getView();
-        CoordinatorLayout.LayoutParams params = (CoordinatorLayout.LayoutParams) v.getLayoutParams();
-        params.gravity = Gravity.TOP;
-        v.setLayoutParams(params);
+        snackBarParams = (CoordinatorLayout.LayoutParams) v.getLayoutParams();
+        snackBarParams.gravity = Gravity.TOP;
+        v.setLayoutParams(snackBarParams);
+
+        TOOLBAR_HEIGHT_IN_PX = convertDpToPixel(56);
```

Comment:
There's a system resource value that encodes the toolbar height, that should update itself depending on whether the phone is vertical or horizontal.  See http://stackoverflow.com/a/13216807/903680